app/src/main/java/com/company/strengthtracker/presentation/test_screen/TestScreen.kt

```diff
@@ -147,7 +147,9 @@ fun ChartHolder(
                     gestureListener = { centroid, panAmount, zoom ->
                         pan = panAmount
                     },
-                    textPaint = textPaint
+                    textPaint = textPaint,
+                    lineSize = 10f
```

Comment:
nit, this is obviously a working area but recommend moving all dimensions like this to the top of the file in companion object then name it a very verbose name like

private const val CHART_LINE_WIDTH_PX = 10f